lib/SILOptimizer/Utils/OwnershipOptUtils.cpp

```diff
@@ -261,8 +261,9 @@ OwnershipLifetimeExtender::copyBorrowAndExtendForRAUW(SILValue newValue,
   while (!frontier.empty()) {
     auto *insertPt = frontier.pop_back_val();
     SILBuilderWithScope frontierBuilder(insertPt);
-    auto *ebi = frontierBuilder.createEndBorrow(insertPt->getLoc(), borrow);
-    auto *dvi = frontierBuilder.createDestroyValue(insertPt->getLoc(), copy);
+    SILLocation loc = RegularLocation(insertPt->getLoc().getSourceLoc());
```

Comment:
My suggestion: lets be careful and just use an autogenerated loc. Otherwise sometimes we can get weird stepping behavior sometimes.